hosted/config/routes.rb

```diff
@@ -3,6 +3,9 @@
 
   resources :inboxes, only: [:show]
   resources :inbound_messages, only: [:create, :show]
+  resources :team_members, param: :public_schedule_slug do
```

Comment:
I am a bit confused by this name, is it the _`public_schedule_slug`_ or is it the _`team_member_slug`_

I.e. is this going to be used to look up the team member or the schedule?

Reply:
It may be that a nested resource is unnecessary here, as we may want to have a schedule that has many team members